components/eamxx/src/physics/ml_correction/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ include(ScreamUtils)
 else()
     message(FATAL_ERROR "pybind11 is missing. Use CMake >= 3.11 or download it")
 endif()
+
+message(STATUS "Python_EXECUTABLE: ${Python_EXECUTABLE}")
```

Comment:
Note: I was originally using this for debugging when there was a conflict between the discovered CMAKE Python and the Pybind python.  Maybe no longer necessary?